datahub-web-react/src/app/ingest/source/executions/ExecutionRequestDetailsModal.tsx

```diff
@@ -120,7 +130,13 @@ export const ExecutionDetailsModal = ({ urn, visible, onClose }: Props) => {
     const resultSummaryText =
         (result && <Typography.Text type="secondary">{getExecutionRequestSummaryText(result)}</Typography.Text>) ||
         undefined;
-    const isOutputExpandable = output.length > 100;
+
+    const recipeJson = data?.executionRequest?.input.arguments?.find((arg) => arg.key === 'recipe')?.value;
+    const recipeYaml = recipeJson && YAML.stringify(JSON.parse(recipeJson), 8, 2);
```

Comment:
do we need error handling here? maybe fallback to raw string if json parsing fails?